### PR TITLE
Doc accounting print

### PR DESCRIPTION
**Pull request by MartinezX21**

### CODE REVIEW

**Comment by mwithi** [comment]
Which OP-XX this pull-request refers to?

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -1024,6 +1024,58 @@ By default, DEBUG is set to no.
...
 
 
+==== 5.1.31 ALLOWMULTIPLEOPENEDBILL
+
+The following table shows the default value and the allowed ones:
+
+[cols=",,",options="header",]
+|===
+|key |default value |allowed values
+|DEBUG |no |yes, no
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]: Fix with correct key value ALLOWMULTIPLEOPENEDBILL

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -1024,6 +1024,58 @@ By default, DEBUG is set to no.
...
+
+
+==== 5.1.32 OPENEDBILLSREPORT
+
+The following table shows the default value and the allowed ones:
+
+[cols=",,",options="header",]
+|===
+|key |default value |allowed values
+|DEBUG |OH023_BillsReportMonth |OH023_BillsReportMonth
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]: Fix with correct key value OPENEDBILLSREPORT

File: doc_admin/AdminManual.adoc [hunk 3]
```diff
@@ -1024,6 +1024,58 @@ By default, DEBUG is set to no.
...
+_*[.underline]#NB#*: An application restart is required in order to see the modification applied._
+
+==== 5.1.33 ALLOWPRINTOPENEDBILL
+
+The following table shows the default value and the allowed ones:
+
+[cols=",,",options="header",]
+|===
+|key |default value |allowed values
+|DEBUG |no |yes, no
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]: Fix with correct key value ALLOWPRINTOPENEDBILL

**Review by mwithi — CHANGES_REQUESTED** [review]
Please make those changes and specify for which OP-XX refers to on Jira

**Comment by MartinezX21** [comment]
> 
> 
> Which OP-XX this pull-request refers to?

This pull request is related to the tasks OP-84, OP-87, OP-88 and OP-90 on Jira.

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (tsognong) [reply doc_admin/AdminManual.adoc]: Ok I see the error

**Comment by MartinezX21** [comment]
@mwithi The changes have been done.

**Review by mwithi — APPROVED** [review]

**Pull request merged**